.github/workflows/update-openapi.yml

```
@@ -0,0 +1,159 @@
+          version: 8
+          run_install: false
+      
+      - name: Install jq for JSON processing
+        run: sudo apt-get install -y jq
+      
+      - name: Fetch and format OpenAPI Spec
+        id: fetch-spec
+        run: |
+          # Create a temp directory
+          mkdir -p tmp
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)